packages/dd-trace/test/appsec/iast/taint-tracking/plugin.express.plugin.spec.js

```diff
@@ -8,6 +8,12 @@ function noop () {}
 describe('Taint tracking plugin sources express tests', () => {
   withVersions('express', 'express', '>=4.8.0', version => {
     prepareTestServerForIastInExpress('in express', version,
+      (expressApp, listener) => {
```

Comment:
why is this tests here, and not as integration test using sandbox to test with the specific multer version?

Reply:
this one is to test the body tainting, I could include it inside [integration](https://github.com/DataDog/dd-trace-js/pull/4781/files#diff-0f99f954cb822f73a211aebfe34a4d7ad9305eeabc5f0c864e5162ca923566b8) test

Reply:
moved